exonum/src/encoding/serialize/mod.rs

```diff
@@ -22,15 +22,24 @@ use encoding::Field;
 use messages::MessageWriter;
 use super::Offset;
 
-/// implement exonum serialization\deserialization based on serde `Serialize`\ `Deserialize`
+/// Implements Exonum serialization\deserialization based on Serde `Serialize`\ `Deserialize`.
 ///
-/// Item should implement:
+/// Under this macro, an item should implement the following entities:
 ///
 /// - `serde::Serialize`
 /// - `serde::Deserialize`
 /// - `exonum::encoding::Field`
 ///
-/// **Beware, this macros probably implement traits in not optimal way.**
+/// Additionally, this macro implements the [`ExonumJson`] trait. `ExonumJson`
```

Comment:
"Additionally" to what? Afaik, `ExonumJson` is the core part of Exonum serialization / deserialization.